api/src/globalShortcut.ts

```diff
@@ -0,0 +1,39 @@
+import { promisified, transformCallback } from './tauri'
+
+/**
+ * register a global shortcut
+ * @param shortcut shortcut definition, modifiers and key separated by "+" e.g. Control+Q
```

Comment:
With the example you provide here, I think it would be better to use CommandOrControl so that it works the same on Mac. See [this](https://www.electronjs.org/docs/api/accelerator#platform-notice) for more details.

Reply:
CommandOrControl isn't supported yet, will be added later

Reply:
Oh ok. Maybe we could use `alt` instead since that is cross-platform and then we don't have to remember to change this later when we do add CommandOrControl.

Reply:
Ctrl works just fine on macOS though

Reply:
It does work, but according to macworld.com:
> On a Mac you use the Command key where on a PC you would use Control (or Ctrl)

So event though it is a key, it's not what a user would expect it to be.

Reply:
there you go

Reply:
Sorry for nitpicking, but I could totally see someone copying that example and then opening an issue complaining that "Tauri keyboard shortcuts are broken reeeeee" :joy:.